helpers/randstr/randstr.go

```diff
@@ -0,0 +1,23 @@
+package randstr
```

Comment:
Maybe add a comment on what this package is all about?

Reply:
Yes

Reply:
Done, 2ceb59f79c32d91a623818ebb927e5e855a6f709